pkg/util/etcdutil/member.go

```diff
@@ -70,6 +70,10 @@ func (m *Member) PeerURL() string {
 	return fmt.Sprintf("%s://%s:2380", m.peerScheme(), m.Addr())
 }
 
+func (m *Member) PVCName() string {
```

Comment:
This is not really etcd related method..
It should be in k8sutil.

Reply:
done